pipelines/cpCore.yaml

```diff
@@ -0,0 +1,280 @@
+      atools.ptcA00PerAmp.unit: "---"
+
+      atools.ptcTurnoffPerAmp: CalibStatisticFocalPlanePlot
+      atools.ptcTurnoffPerAmp.produce.plot.addHistogram: true
+      atools.ptcTurnoffPerAmp.produce.plot.zAxisLabel: "PTC Turnoff (ADU)"
+      atools.ptcTurnoffPerAmp.quantityKey: PTC_PTC_TURNOFF
+      atools.ptcTurnoffPerAmp.unit: ADU
+
+      atools.ptcMaxPerAmp: CalibStatisticFocalPlanePlot
+      atools.ptcMaxPerAmp.produce.plot.addHistogram: true
+      atools.ptcMaxPerAmp.produce.plot.zAxisLabel: "PTC Max Raw Mean Flux (ADU)"
+      atools.ptcMaxPerAmp.quantityKey: PTC_PTC_MAX_RAW_MEANS
+      atools.ptcMaxPerAmp.unit: ADU
+
+      atools.ptcRowMeanVarianceSlopePerAmp: CalibStatisticFocalPlanePlot
+      atools.ptcRowMeanVarianceSlopePerAmp.produce.plot.addHistogram: true
+      atools.ptcRowMeanVarianceSlopePerAmp.produce.plot.zAxisLabel: "PTC Row-Mean-Variance Slope (e-)"
+      atools.ptcRowMeanVarianceSlopePerAmp.quantityKey: PTC_PTC_ROW_MEAN_VARIANCE_SLOPE
+      atools.ptcRowMeanVarianceSlopePerAmp.unit: electron
+
+      python: |
+        from lsst.analysis.tools.atools import *
+
+  analyzePtcDetCore:
+    class: lsst.analysis.tools.tasks.VerifyCalibDetectorTask
+    config:
+      connections.outputName: cpPtcDetCore
+      connections.data: verifyPtcDetResults
+
+      atools.ptcPlot: CalibAmpScatterTool
+      atools.ptcPlot.prep.panelKey: amplifier
+      atools.ptcPlot.prep.dataKey: PTC_PTC_RAW_MEANS
+      atools.ptcPlot.prep.quantityKey: PTC_PTC_RAW_VARIANCE
+      atools.ptcPlot.produce.plot.xlabel: "Exposure Pair Flux (ADU)"
+      atools.ptcPlot.produce.plot.ylabel: "Exposure Pair Variance (ADU^2)"
+      python: |
+        from lsst.analysis.tools.atools import *
+
+  analyzeDefectCore:
+    class: lsst.analysis.tools.tasks.VerifyCalibAnalysisTask
+    config:
+      connections.outputName: cpDefectCore
+      connections.data: verifyDefectResults
+
+      # TODO: DM-43880 Sort out defects
+      atools.defectsHotPixels: CalibStatisticFocalPlanePlot
+      atools.defectsHotPixels.produce.plot.addHistogram: true
+      atools.defectsHotPixels.produce.plot.zAxisLabel: "Number of Hot Pixels"
+      atools.defectsHotPixels.quantityKey: DEFECTS_DEFECTS_DIST_N_HOT
+      atools.defectsHotPixels.unit: "---"
+
+      # atools.defectsColdPixels: CalibStatisticFocalPlanePlot
+      # atools.defectsColdPixels.produce.plot.addHistogram: true
+      # atools.defectsColdPixels.produce.plot.zAxisLabel: "Number of Cold Pixels"
+      # atools.defectsColdPixels.quantityKey: DEFECTS_DEFECTS_DIST_N_COLD
+      # atools.defectsColdPixels.unit: "---"
+      # atools.defectsBadColumns: CalibStatisticFocalPlanePlot
+      # atools.defectsBadColumns.produce.plot.addHistogram: true
+      # atools.defectsBadColumns.produce.plot.zAxisLabel: "Number of Hot Pixels"
+      # atools.defectsBadColumns.quantityKey: DEFECTS_DEFECTS_DIST_N_BAD_COLUMNS
+      # atools.defectsBadColumns.unit: "---"
+      python: |
```

Comment:
Should all these commented out bits be here?

Reply:
I've left commented blocks that have associated JIRA tickets.  I think in most of these cases, the commented code will work once those tickets merge (and hopefully un-comment these blocks).